contrib/lib/tests/compressed.rs

```diff
@@ -0,0 +1,254 @@
+#![feature(decl_macro, proc_macro_hygiene)]
+
+#[macro_use]
+extern crate rocket;
+extern crate rocket_contrib;
+
+#[cfg(all(feature = "brotli_compression", feature = "gzip_compression"))]
+mod compressed_tests {
+    extern crate brotli;
+    extern crate flate2;
+
+    use rocket::http::hyper::header::{ContentEncoding, Encoding};
+    use rocket::http::Status;
+    use rocket::http::{ContentType, Header};
+    use rocket::local::Client;
+    use rocket::response::Response;
+    use rocket::routes;
+    use rocket_contrib::compression::Compressed;
+
+    use std::io::Cursor;
+    use std::io::Read;
+
+    use self::flate2::read::{GzDecoder, GzEncoder};
+
+    const HELLO: &str = r"This is a message to hello with more than 100 bytes \
+        in order to have to read more than one buffer when gzipping. こんにちは!";
+
+    #[get("/")]
+    pub fn index() -> Compressed<String> {
+        Compressed::new(String::from(HELLO))
+    }
+
+    #[get("/font")]
+    pub fn font() -> Compressed<Response<'static>> {
```

Comment:
`Compressed<Content<&'static str>>` and `Compressed::new(Content(ContentType::WOFF, HELLO))` is much shorter and a better example to show anyone reading the code. Similar for `image`.